helm/methode-article-mapper/templates/deployment.yaml

```diff
@@ -0,0 +1,52 @@
+##this is an example deployment.yaml that should be customized in order to meet the configuration for app k8s deployment
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Values.service.name }} 
+  labels:
+    chart: "{{ .Chart.Name | trunc 63 }}"
+    chartVersion: "{{ .Chart.Version | trunc 63 }}" 
+    visualize: "true" 
+    app: {{ .Values.service.name }} 
+spec:
+  replicas: {{ .Values.replicaCount }} 
+  selector: 
+    matchLabels:
+      app: {{ .Values.service.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.service.name }}
+        visualize: "true" 
+    spec:
+      containers: 
+      - name: {{ .Values.service.name }} 
+        image: "{{ .Values.image.repository }}:{{ .Chart.Version }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env:
+        - name: DOCUMENT_STORE_API_URL
+          value: "document-store-api:8080"
+        - name: CONCORDANCE_API_URL
+          value: "public-concordances-api:8080"
+        - name: KAFKA_PROXY_URL
+          valueFrom:
+            configMapKeyRef:
+              name: global-config
+              key: kafka.proxy.url
+        ports: 
+        - containerPort: 8080
+        - containerPort: 8081
+        livenessProbe: 
```

Comment:
Please set the liveness probe delay to be smaller than the readiness probe delay.

Reply:
Done.